build/.vsts-pr.yml

```diff
@@ -4,6 +4,8 @@
 name: pr$(system.pullRequest.pullRequestNumber)-$(Date:yyyyMMdd)$(Rev:-r)
 variables:
   buildConfiguration: 'Release'
+  azureSubscriptionEndpoint: 'docker-test'
+  azureContainerRegistry: 'ossdockertest.azurecr.io'
```

Comment:
Should we use HealthPlatformRegistry for this? It has anonymous pull enabled (i.e. public access).

Reply:
That's a good idea for the final repo. Right now I'm trying to get it to work, so I think a test account is best.